data/strings/types_strings.txt

```diff
@@ -17279,38 +17291,14 @@
     zh-Hant = 理髮師
 
   [type.shop.hardware]
-    en = Hardware Store
-    ar = متجر عدد وأدوات
-    cs = Železářství
-    da = Isenkræmmer
-    de = Eisenwarengeschäft
-    el = Είδη κιγκαλερίας
-    es = Ferretería
-    eu = Burdindegia
-    fa = فروشگاه
-    fi = Rautakauppa
-    fr = Quincaillerie
-    hu = Vaskereskedés
-    id = Toko perangkat keras
-    it = Ferramenta
-    ja = 工具店
-    ko = 철물점
-    mr = हार्डवेअर दुकान
-    nb = Jernvareforretning
-    nl = Ijzerhandel
-    pl = Sklep narzędziowy
-    pt = Loja de ferragens
-    pt-BR = Loja de ferragens
-    ro = Magazin de bricolaj
-    ru = Хозяйственный
-    sk = Železiarstvo
-    sv = Järnhandel
-    th = ร้านขายฮาร์ดแวร์
-    tr = Hırdavatçı
-    uk = Будматеріали
-    vi = Cửa hàng phần cứng
-    zh-Hans = 硬件店
-    zh-Hant = 五金行
+    ref = type.shop.doityourself
+
+  [type.shop.health_food]
+    en = Health Food Shop
+    ru = Магазин здоровой еды
+
+  [type.shop.houseware]
+    ref = type.shop.doityourself
```

Comment:
For some languages the translation will be the same, but e.g. `en = Houseware` will fit better than "Hardware Store".